ToDoList-frontend/src/environments/environment.ts

```diff
@@ -5,6 +5,8 @@
 export const environment = {
   production: false,
   localhostUrl: 'https://localhost:7247/api/',
+
```

Comment:
Nie robmy tak :/ Mysle ze ten projekt jest dobry do wzorowania sie na poczatek: https://github.com/gothinkster/angular-realworld-example-app/tree/main/src/app/core/interceptors Jest tam przyklad interceptora do base url i do errorow. Możesz sklonowac, odpalic, zobaczyc jak dziala. Wystarczy jedynie lekko pozmieniac. Dodajmy te interceptory i przerobmy serwisy w nastepnym PR ;)

Reply:
Oki to na razie przeniosłem do projektu api.interceptor.ts oraz error.interceptor.ts(token.interceptor.ts na razie pozostawiem zakomentowany). W app.module.ts również zakomentowałem providers dla tych 2 interceptorów. Jeśli dobrze to rozumiem, to po włączeniu ich w providers cały Angular już pracuje na nich wykorzystując zahardkodowany przeze mnie url w api interceptor oraz końcówkę którą hm trzebaby dopasowywać w serwisach? 
Ogólnie wydaje mi się, że to będzie dużo lepsze niż sztywne ścieżki w environment.ts, gdyż będzie możliwość w jednym miejscu globalnie dostosować zachowanie przy requestach http.
Daj znać co myślisz :)